specs/batcher.md

```diff
@@ -4,20 +4,18 @@
 The batch submitter, also referred to as the batcher, is the entity submitting the L2 sequencer data to L1,
 to make it available for verifiers.
 
-[derivation-spec]: ./derivation.md
```

Comment:
Does this format not work?

Reply:
no it didn't render, at least for github

Reply:
I think this might actually be a bug with github markdown rendering?

See: https://github.com/ethereum-optimism/optimism/blob/develop/specs/batcher.md

For example, these all render:

[derivation-spec]: ./derivation.md

Explicit: [derivation-spec](derivation-spec)
Explicit2: [derivation-spec](./derivation-spec.md)

Aliased: [derivation-spec][derivation-spec]

Referenced: [derivation-spec]

Reply:
Uh oh, isn't the aliased format used predominantly throughout the specs?

Reply:
Ah right - I fixed it to be aliased properly. Believe it was the `-` prefix and relative filepath that borked it.

Reply:
Ah, good catch, but I think it'd be better if the hyphen went into the alias instead of the text itself.